backend/services/ephemeris_service.py

```
@@ -21,38 +31,65 @@
+    "moon": swe.MOON if SWE_AVAILABLE else 1,
+    "mercury": swe.MERCURY if SWE_AVAILABLE else 2,
+    "venus": swe.VENUS if SWE_AVAILABLE else 3,
+    "mars": swe.MARS if SWE_AVAILABLE else 4,
+    "jupiter": swe.JUPITER if SWE_AVAILABLE else 5,
+    "saturn": swe.SATURN if SWE_AVAILABLE else 6,
+    "uranus": swe.URANUS if SWE_AVAILABLE else 7,
+    "neptune": swe.NEPTUNE if SWE_AVAILABLE else 8,
+    "pluto": swe.PLUTO if SWE_AVAILABLE else 9,
 }
 
 def _julian_day(dt: datetime) -> float:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)